meta/SaiSerialize.cpp

```diff
@@ -2626,6 +2626,66 @@ std::string sai_serialize_redis_communication_mode(
     }
 }
 
+
+json sai_serialize_stat_capability(
+        _In_ const sai_stat_capability_t& stat_capability,
+        _In_ const sai_enum_metadata_t* meta)
+{
+    SWSS_LOG_ENTER();
+
+    json j;
+
+    j["stat_enum"] = sai_serialize_enum(stat_capability.stat_enum, meta);
+
+    json arr = json::array();
+
+    for (uint32_t i = 0; i < sai_metadata_enum_sai_stats_mode_t.valuescount; ++i)
+    {
+	if (stat_capability.stat_modes & sai_metadata_enum_sai_stats_mode_t.values[i])
+	{
+            json item = sai_serialize_enum(sai_metadata_enum_sai_stats_mode_t.values[i],
+                            &sai_metadata_enum_sai_stats_mode_t);
+
+            arr.push_back(item);
+	}
+    }
+
+    j["stat_modes"] = arr;
```

Comment:
you are serializing modes as array, not sure if this should be in this way or just and integer value